slotfills/fm-gutenberg-fields.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { withSelect } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+
+import MetaBox from './sections/meta-box';
+
+const FmGutenbergFields = (post) => {
+  const { post: { fm_gutenberg_fields: fmFields } } = post;
+  if (!fmFields) {
```

Comment:
Should we check for `.length` here?

Reply:
actually removing this if statement, because since it's in the post object we never have to wait for it to load.